### ISSUE TITLE
Are move refactorings supported?

### ISSUE THREAD

**Issue by jgarte** [pappasam/jedi-language-server#209]
Hi,

Are [move refactorings](https://www.jetbrains.com/help/pycharm/move-refactorings.html) supported in `jedi-language-server`?

**Comment by pappasam** [comment]
Yes. If I understand that link correctly, they're called ["rename".](https://github.com/pappasam/jedi-language-server#language-features).

**Comment by jgarte** [comment]
I was a bit confused of how to achieve this with eglot and emacs as I've used [eglot-rename](https://github.com/joaotavora/eglot/blob/master/eglot.el#L2960) before but just to rename a variable but not to move a module to a different directory in the project's source tree while renaming all the imports. I'll give it another try. Thanks for the clarification and help.

**Comment by jgarte** [comment]
@pappasam Hi, how can I test that this works in emacs with eglot. There is no documentation on how to run move refactorings but I would really like to start using them in Emacs. Any suggestions for how to debug this? I've had no luck so far.
